spec/enumerable_spec.rb

```diff
@@ -0,0 +1,296 @@
+      end
+    end
+  end
+
+  describe '#my_all?' do
+    context 'when no block is given' do
+      it 'items are not false or nil' do
+        expect(array_string.my_all?).to eql(array_string.all?)
+      end
+      it 'items are false or nil' do
+        expect(array_with_nil.my_all?).to eql(array_with_nil.all?)
+      end
+      it 'items are not given' do
+        expect([].my_all?).to eql([].all?)
+      end
+    end
+    context 'when block is given' do
+      it 'items are not false or nil' do
+        expected = array_string.all? { |word| word.length >= 3 }
+        expect(array_string.my_all? { |word| word.length >= 3 }).to eql(expected)
+      end
+      it 'items are not false or nil' do
+        expected = array_string.all? { |word| word.length >= 4 }
+        expect(array_string.my_all? { |word| word.length >= 4 }).to eql(expected)
+      end
+    end
+    context 'when regular expression is given' do
+      it 'items are not false or nil' do
+        expected = array_string.all?(/t/)
+        expect(array_string.my_all?(/t/)).to eql(expected)
+      end
+    end
+    context 'when class is given' do
+      it 'items are not false or nil' do
+        expected = array_with_invalid_numeric.all?(Numeric)
+        expect(array_with_invalid_numeric.my_all?(Numeric)).to eql(expected)
+      end
+    end
+  end
+
+  describe '#my_any?' do
+    context 'when no block is given' do
+      it 'returns false' do
+        expect([].my_any?).to eql(false)
+      end
+      it 'items are false or nil' do
+        expect(array_with_nil.my_any?).to eql(array_with_nil.any?)
+      end
+    end
+    context 'when block is given' do
+      it 'items are not false or nil' do
+        expected = array_string.any? { |word| word.length >= 3 }
+        expect(array_string.my_any? { |word| word.length >= 3 }).to eql(expected)
+      end
+      it 'items are not false or nil' do
+        expected = array_string.any? { |word| word.length >= 4 }
+        expect(array_string.my_any? { |word| word.length >= 4 }).to eql(expected)
+      end
+    end
+    context 'when regular expression is given' do
+      it 'items are not false or nil' do
+        expected = array_string.any?(/t/)
+        expect(array_string.my_any?(/t/)).to eql(expected)
+      end
+    end
+    context 'when class is given' do
+      it 'items are false or nil' do
+        expected = array.any?(Numeric)
+        expect(array.my_any?(Numeric)).to eql(expected)
+      end
+      it 'items are not false or nil' do
+        expected = array_with_nil.any?(Numeric)
+        expect(array_with_nil.my_any?(Numeric)).to eql(expected)
+      end
+    end
+  end
+
+  describe '#my_none?' do
+    context 'when no block is given' do
+      it 'returns false' do
+        expect([].my_none?).to eql(true)
+      end
+      it 'items are nil' do
+        expect(array_with_one_nil.my_none?).to eql(array_with_one_nil.none?)
+      end
+      it 'items are false or nil' do
```

Comment:
Add info related to the task covered. Same as above.

Reply:
Fixed, thanks for the suggestion